modules/olm-arch-catalog-operator.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding_olm/olm-arch.adoc
+
+[id="olm-arch-catalog-operator_{context}"]
+= Catalog Operator
+
+The Catalog Operator is responsible for resolving and installing CSVs and the
+required resources they specify. It is also responsible for watching
+CatalogSources for updates to packages in channels and upgrading them
+(optionally automatically) to the latest available versions.
+
+A user who wishes to track a package in a channel creates a Subscription
+resource configuring the desired package, channel, and the CatalogSource from
+which to pull updates. When updates are found, an appropriate InstallPlan is
+written into the namespace on behalf of the user.
+
+Users can also create an InstallPlan resource directly, containing the names of
+the desired CSV and an approval strategy, and the Catalog Operator creates an
+execution plan for the creation of all of the required resources. After it is
+approved, the Catalog Operator creates all of the resources in an InstallPlan;
+this then independently satisfies the OLM Operator, which proceeds to install
+the CSVs.
+
+.Catalog Operator workflow
+* Has a cache of CRDs and CSVs, indexed by name.
```

Comment:
This workflow is also out of date (it describes the catalog operator when there were only ConfigMap catalogs)

I would say "Has a connection to each CatalogSource in the cluster"

Reply:
Updated this bullet, presuming the following bullets are still accurate.